modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-azure/src/test/resources/log4j2-test.xml

```diff
@@ -12,6 +12,7 @@
   </Appenders>
   <Loggers>
     <Logger name="org.nuxeo" level="info" />
+    <Logger name="org.nuxeo.ecm.blob.azure" level="debug" />
```

Comment:
Did you really want to commit this? It's bad practice to have verbose logs during tests.

Reply:
Yes, it's to have the interesting logs from `AzureFileStorage` and they are only appended to the log file, not the console.